frontend/src/console/components/NoticeBoard/styles.scss

```diff
@@ -40,6 +40,17 @@
 }
 
 .notice-board-heading {
-  font-size: 25px;
+  font-family: $chivo-font;
```

Comment:
Has the proper font family been settled out? I've seen your comment in Jira, but I could only find the Zeplin designs, which contained `Montserrat`, hence the question.

Reply:
Yep, it's Chivo. The one shown in the mockups is not correct, as discussed [here](https://tasks.opencraft.com/browse/BB-3500?focusedCommentId=187842&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-187842)